Description: We don't ship oui.txt, don't try to install
 Debian has an ieee-data package which releases us from shipping
 IEEE's oui.txt. We need to tweak GNUmakefile so it doesn't
 attempt to install the file, or make install will fail.
Author: Octavio Alvarez <alvarezp@alvarezp.com>
Bug-Debian: http://bugs.debian.org/739202
Forwarded: not-needed

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -131,8 +131,8 @@
 	install -m0644 data/ipv6toolkit.conf $(ETCPATH)
 
 	# Install the IEEE OUI database
-	install -m0755 -d $(DATAPATH)
-	install -m0644 data/oui.txt $(DATAPATH)
+	#install -m0755 -d $(DATAPATH)
+	#install -m0644 data/oui.txt $(DATAPATH)
 
 	# Install the port numbers database
 	install -m0644 data/service-names-port-numbers.csv $(DATAPATH)
